src/FSharpPlus/Operators.fs

```diff
@@ -757,6 +778,13 @@ module Operators =
     /// <exception cref="System.ArgumentNullException">Thrown when the input collection is null.</exception>
     let inline sortByDescending (projection: 'T->'Key when 'Key : comparison) (source: '``Collection<'T>``) : '``Collection<'T>`` = SortByDescending.Invoke projection source
 
+    /// Splits a given ordered collection at each of the given sub-ordered collections
+    ///
+    ///     > "asdf" |> split ["s"];;
+    ///        val it : string list = ["a"; "df"]
+    ///
+    ///     > [1;2;3;4;5;6] |> split [ [2]; [5] ];;
+    ///        val it : int list list = [[1]; [3; 4]; [6]]
```

Comment:
I'm not 100% sure including sample code in XML docs is common practice.
Though the sample is very illustrative.

Reply:
there are tags for code when documenting c#, these should be available in f# as well : 

https://docs.microsoft.com/en-us/dotnet/csharp/codedoc#example

Reply:
Thankyou for the reference! I thought it was all FSharp.Formatting that was reading and implementing these, and even wondered why you used xml, when that is supported markdown too. 
(this is my newness to .net showing).

I will read that link about codedoc and use these in future.